### PR TITLE
Make it pretty

### PR DESCRIPTION
**Pull request by majornorth**
I had to resolve some merge conflicts with the style changes I did (and also an update to the views.js). This branch doesn't load, but throws a 500 error, so I'll need your help resolving that.

### CODE REVIEW

**Comment by majornorth** [comment]
@ezarowny actually, I checked out master and that doesn't load for me either. So the merge conflicts had nothing to do with the 500 error.

**Comment by majornorth** [comment]
@ezarowny okay, I saw your Hipchats from yesterday and deleted my db and re-ran migrations. I don't get the 500 error now, but the form input doesn't render.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,7 @@
 # hey-look-a-todo-app
-Let's make a ToDo app with @stewartmccoy because he's a real radical dude!
+A Todo app built with Django, Backbone.js, and MongoDB. Developed by [Eric Zarowny](http://www.linkedin.com/in/ezarowny/en) and [Stewart McCoy](http://www.linkedin.com/in/stewartmccoy/en).
```
Comment (ezarowny) [inline README.md]: It's just running SQLite right now :P. We'll probably end up with Postgres at some point.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,7 @@
...
+
+To run locally:
+
+1. `workon todo`
```
Comment (ezarowny) [inline README.md]: This only works if you have virtualenv installed and named your env 'todo'. Maybe too specific to our computers? I could add more generalized directions or we should probably leave it out. Thoughts?

File: README.md (hunk 1 above)
Comment (ezarowny) [inline README.md]: Oh, we are using Marionette!

**Comment by majornorth** [comment]
@ezarowny +1?

**Comment by ezarowny** [comment]
:+1:

**Pull request merged**